homeassistant/components/goalzero/const.py

```diff
@@ -25,6 +48,26 @@
     "inputDetected": ["Input Detected", DEVICE_CLASS_POWER, None],
 }
 
+SENSOR_DICT = {
```

Comment:
Maybe use a typed dict or dataclass to make more clear what each position means?